### ISSUE TITLE
Add missing tags (<mark>, <cite>, <dfn> ...)

### ISSUE THREAD

**Issue by finex** [Roslovets-Inc/strapi-plugin-ckeditor5#14]
Hi, currently this plugin doesn't allow to use a lot of HTML tags, or at least I've not found any documentation.
I'm talking about:

- mark
- cite
- dfn
- abbr
- dd / dl / dt
- del
- ins
- kbd
- samp

Would be possible to add them? I  use mark / cite / dfn and abbr, but other people could find useful the other common tags.

Thanks for your attention.

**Comment by roslovets** [comment]
Hi, did you try to insert HTML Snippet?

You can find it at the bottom of [Live Demo](https://roslovets-inc.github.io/ckeditor5-build-strapi-wysiwyg/sample.html) example.

Example for `<mark>`:
![image](https://user-images.githubusercontent.com/21337619/115929774-02419e00-a491-11eb-8856-2cb21cddbf5a.png)

**Comment by roslovets** [comment]
Use this button
![image](https://user-images.githubusercontent.com/21337619/115929953-4fbe0b00-a491-11eb-9109-6a86d9782b9c.png)

**Comment by finex** [comment]
Yes, I did, but it's not the right way because that button adds an extra `<div>` outside the code.

**Comment by roslovets** [comment]
> adds an extra `<div>` outside the code.

Is it real problem in your case? Because I don't see any simple option to omit this parent div. Seems like you have to introduce any new element as new toolbar button. [This](https://github.com/centaur54dev/ckeditor5-custom-element) can help.

**Comment by finex** [comment]
> > adds an extra `<div>` outside the code.
> 
> Is it real problem in your case? Because I don't see any simple option to omit this parent div. Seems like you have to introduce any new element as new toolbar button. [This](https://github.com/centaur54dev/ckeditor5-custom-element) can help.

Yes, it's a problem because it adds unwanted markup breaking up styling. I will check the ckeditor5-custom-element. Thank you. 

Anyway it could be a very useful feature on your plugin, instead of adding extra plugins :-)
